hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/Store.java

```diff
@@ -299,4 +299,11 @@
    *         if you try to set a configuration.
    */
   Configuration getReadOnlyConfiguration();
+
+  /**
+   * @return the identifier string of a store, formatted to encodeRegionName:familyName
+   */
+  default String getIdentifier() {
```

Comment:
The name seems too general...

Could we use just identity hash map to put the store instance in it directly?

Reply:
I used this name in the log printing, I think the store info is useful in the compaction logs...